admin/scripts/modules/uzivatel.php

```diff
@@ -153,9 +154,13 @@
     $zbyva    = $r['zbyva'] === null
         ? '&infin;'
         : $r['zbyva'];
-    $moznosti .= '<option value="' . $r['id_predmetu'] . '"' . ($r['zbyva'] > 0 || $r['zbyva'] === null
-            ? ''
-            : ' disabled') . '>' . $r['nazev'] . ' (' . $zbyva . ') ' . $r['cena'] . '&thinsp;Kč</option>';
+
+    // úplně odstraníme logiku pro 'disabled', aby šlo prodávat i při zbyva ≤ 0
+    $moznosti .= '<option value="' . $r['id_predmetu'] . '">'
```

Comment:
Tohle se dá zčitelnit na
```suggestion
    $moznosti .= "<option value='{$r['id_predmetu']}>'>{$r['nazev']} ($zbyva) {$r['cena']}&thinsp;Kč</option>";
```
(pokud zkousneme že do HTML posíláme jednoduché uvozovky)